### ISSUE TITLE
Keyboard navigation (a11y)

### ISSUE THREAD

**Issue by adufr** [gitify-app/gitify#1381]
### 📝 Provide a description of the new feature

I'd like to be able to navigate through Gitify by only using my keyboard. (Following #995)

### ➕ Additional Information

Most elements are missing a `focus` style, which makes the keyboard navigation impossible.

This feature would be very useful for both power-users and/or users with disabilities.

Let's make Gitify more inclusive! 😁

**Comment by adufr** [comment]
I haven't done any PoC for now, but I'd like to start contributing again so I might try and time to work on this -- if you approve the idea :)

**Comment by setchy** [comment]
> I haven't done any PoC for now, but I'd like to start contributing again so I might try and take some time to work on this -- if you approve the idea :)


Absolutely worth exploring, @adufr

**Comment by setchy** [comment]
Closing this (for now) as our migration to GitHub Primer Components as part of the upcoming [v6.0.0 milestone](https://github.com/gitify-app/gitify/milestone/37) should address the majority of accessibility items.

Of course, quite happy to reopen if there are gaps post v6 launch.
